runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPlugin.java

```diff
@@ -103,12 +102,6 @@ protected void shutDown() throws Exception
 		clientToolbar.removeNavigation(navButton);
 	}
 
-	@Subscribe
-	public void onPluginChanged(PluginChanged event)
```

Comment:
Why was this removed?

Reply:
I removed the event handler as it was calling Configpanel::refreshPluginList() whenever a single plugin was either enabled or disabled. (This is also the only time its called outside of the ConfigPanel ctor)

```java
void refreshPluginList()
{
	// update enabled / disabled status of all items
	pluginList.forEach(listItem ->
	{
		final Plugin plugin = listItem.getPlugin();
		if (plugin != null)
		{
			listItem.setPluginEnabled(pluginManager.isPluginEnabled(plugin));
		}
	});

	if (showingPluginList)
	{
		openConfigList();
	}
}
```

As you can see its just rebuilding the whole plugin list. Unless this is some hacky fix for an unknown issue, it seems redundant to me. And rebuilding the plugin list prevents me from performing my animations. 

I guess if you wanted to keep this, I could add a parameter to the event that lets the refreshPluginList() method know to fire an animation for the updated plugin?

Reply:
It is here so the panel will update when plugin is toggled outside of config panel (so, for example GPU plugin, or logging in/out of rl account)

Reply:
@deathbeam Oh right, thanks! I will definitely need to change a lot of my impl then

Reply:
@deathbeam I can't seem to find any code that toggles the state of plugins outside of actually clicking the button itself, can you point me in the right direction? I also think I may have found a bug with what you are describing